espnet2/bin/s2t_inference.py

```diff
@@ -575,6 +575,12 @@ def decode_long(
         text_prev = init_text
         while offset < len(speech):
             logging.info(f"Current start time in seconds: {offset / fs:.2f}")
+            if offset + segment_len > len(speech) and len(segment) / fs < 0.2:
+                logging.warning(
+                    f"Skip the last clip as it's too short: {len(segment)/ fs:.2f}s"
```

Comment:
Keep the style consistent. Now there is a space after `/` but no space before it.

Reply:
We can change "clip" to "segment" or "chunk"